testkit/src/main/scala/org/bitcoins/testkit/util/BitcoinSAsyncTest.scala

```diff
@@ -50,6 +50,15 @@ trait BaseAsyncTest
 
   override lazy val timeLimit: Span = 5.minutes
 
+  lazy val isLinux: Boolean =
```

Comment:
nit: Can you move these into `BitcoinSUtil` so we can use them elsewhere too?

Reply:
Actually you should probably just use `scala.util.Properties.isMac` or `Properties.isLinux` etc